javascript/features/death_match/death_match_stats.js

```diff
@@ -0,0 +1,32 @@
+// Copyright 2020 Las Venturas Playground. All rights reserved.
+// Use of this source code is governed by the MIT license, a copy of which can
+// be found in the LICENSE file.
+
+export class DeathMatchStats {
+    constructor() {
+        this.kills_ = 0;
+        this.deaths_ = 0;
+        this.damage_ = 0;
+
+        this.bulletsHit_ = 0;
+        this.bulletsMissed_ = 0;
+    }
```

Comment:
For classes like this, I'd be inclined to make them POD w/ extra functionality, like:

```javascript
export class DeathMatchStats {
    kills = 0;
    deaths = 0;

    damage = 0;

    bulletsHit = 0;
    bulletsMissed = 0;

    // Gets the kill/death ratio during the match.
    get kdRatio() {
        return this.deaths === 0 ? this.kills : Math.round(this.kills / this.deaths * 100) / 100;
    }

    // Gets their accuracy during the match.
    get accuracy() {
        const bulletsShot = this.bulletsHit + this.bulletsMissed;
        return bulletsShot === 0 ? 100 : Math.round((this.bulletsHit / bulletsShot * 100));
    }
}
```

...and then just change the variables, so instead of calling `addKill()` you'd just to `stats.kills++`;

Reply:
The reason I choose to use `addKill()` instead of making kills "public" is because I saw no reason why we would take kills or award multiple kills at once. But I don't think this is of that much importance indeed :+1:

Also was not aware that doing `kills = 0;` is a valid thing to do in JS 👍